src/app/payment/PaymentSubmitButton.tsx

```diff
@@ -98,8 +111,15 @@ export default withCheckout(({ checkoutState }) => {
         },
     } = checkoutState;
 
+    const ppsdkFeatureOn = Boolean(
+        checkoutService.getState()
+            .data.getConfig()
+            ?.checkoutSettings.features['PAYMENTS-6806.enable_ppsdk_strategy']
+    );
+
     return {
         isInitializing: isInitializingCustomer() || isInitializingPayment(),
         isSubmitting: isSubmittingOrder(),
+        ppsdkFeatureOn,
```

Comment:
minor, for consistency with other variables, maybe name it `isPpsdkFeatureEnabled`